File: QuanLyPhongKham.Client/src/interfaces/responses/NhanVienResponse.js
/**
 * @typedef {Object} NhanVienResponse
 * @property {string} maNV - Employee ID
 * @property {string} hoTen - Full name
 * @property {string} email - Email
 * @property {string} soDienThoai - Phone number
 * @property {string} role - Role
 * @property {string} password - Password
 */

// password không trả về frontend
export const createNhanVienResponse = (
	maNV = '',
	hoTen = '',
	email = '',
	soDienThoai = '',
	role = ''
) => ({
	maNV,
	hoTen,
	email,
	soDienThoai,
	role,
});

File: QuanLyPhongKham.Server/src/QuanLyPhongKham.DTOs/Responses/NhanVienResponse.cs
using System;

namespace QuanLyPhongKham.DTOs.Responses
{
    public class NhanVienResponse
    {
        public Guid MaNV { get; set; }
        public string HoTen { get; set; }
        public string? Email { get; set; }
        public string? SoDienThoai { get; set; }
        public string Role { get; set; }
    }
}
